ledger/participant-integration-api/src/test/lib/scala/platform/store/backend/StorageBackendTestsIngestion.scala

```diff
@@ -93,20 +94,24 @@ private[backend] trait StorageBackendTestsIngestion
     partiesAfterLedgerEndUpdate should not be empty
   }
 
-  private val NumberOfUpsertPackagesTests = 100
+  private val NumberOfUpsertPackagesTests = 30
   it should s"safely upsert packages concurrently ($NumberOfUpsertPackagesTests)" in withConnections(
     2
   ) { connections =>
     import scala.concurrent.ExecutionContext.Implicits.global
 
+    val List(connection1, connection2) = connections
+    def packageFor(n: Int): DbDto.Package =
+      dtoPackage(offset(n.toLong))
+        .copy(
+          package_id = s"abc123$n",
+          _package = Random.nextString(Random.nextInt(200000) + 200).getBytes,
```

Comment:
this pumps up the unit test so it fails for the oracle bug (with the previous implementation)

Reply:
`package_size` is not following the actual payload size. This particular field is currently just written verbatim to the database and no logic is attached to it. It is however asking for future trouble.
I suggest augmenting `dtoPackage` function to deal with this

Reply:
From the DB schema and the StorageBackend point of view the package_size is just dead weight, no meaning
I think it has relevance only for the other side of the Ledger API, and we populate it in the UpdateToDbDto:
`package_size = archive.getPayload.size.toLong,`